internal/statestore/backfill.go

```diff
@@ -200,6 +208,48 @@ func (rb *redisBackend) UpdateBackfill(ctx context.Context, backfill *pb.Backfil
 	return nil
 }
 
+// CleanupBackfills removes expired backfills
+func (rb *redisBackend) CleanupBackfills(ctx context.Context) error {
```

Comment:
Sorry for asking this here, and not in backfill.go, but I think it is related closely enough:

Why do we expire Backfills at 80% of the pendingReleaseTimeout? I think it isn't very clear that setting pendingReleaseTimeout will have an effect on when Backfills are deleted. I think it would be better for Backfills to have their own explicit expiration timeout in the config, and to update the ticket release process to disassociate the ticket from a Backfill, if it was associated with one.

Reply:
We had a conversation with @Laremere in one of the PRs thread and agreed on using this 80%. 
@aLekSer we do plan to add this value to a separate config parameter, don't we?

Reply:
Ok. Thanks for discussing it with @Laremere. If this was the agreed-upon approach, then I'm fine with it.